internal/journal/voljournal.go

```diff
@@ -805,7 +806,8 @@ func (conn *Connection) StoreAttribute(ctx context.Context, pool, reservedUUID,
 
 // StoreGroupID stores an groupID in omap.
 func (conn *Connection) StoreGroupID(ctx context.Context, pool, reservedUUID, groupID string) error {
-	err := conn.StoreAttribute(ctx, pool, reservedUUID, conn.config.csiGroupIDKey, groupID)
+	err := setOMapKeys(ctx, conn, pool, conn.config.namespace, conn.config.cephUUIDDirectoryPrefix+reservedUUID,
+		map[string]string{conn.config.csiGroupIDKey: groupID})
```

Comment:
@nixpanic i think this change is not required because `StoreAttribute` is doing the same functionality.

Reply:
The difference is that `StoreAttribute` adds the `csi.` prefix, that makes it `csi.csi.groupid` as key, and reading it back is done with `csi.groupid`.